js/core/gameSim.js

```diff
@@ -164,63 +161,70 @@ define(["lib/underscore", "util/helpers", "util/random"], function (_, helpers,
         return out;
     };
 
-    /**
-     * Simulate this.numPossessions possessions.
-     *
-     * To simulate regulation or overtime, just set this.numPossessions to the appropriate value and call this function.
-     *
-     * @memberOf core.gameSim
-     */
-    GameSim.prototype.simPossessions = function () {
-        var i, outcome, substitutions;
+    GameSim.prototype.simRegulation = function () {
+        var quarter;
 
         this.o = 0;
         this.d = 1;
+        quarter = 1;
 
-        i = 0;
-        while (i < this.numPossessions * 2) {
-            // Keep track of quarters
-            if ((i * this.dt > 12 && this.team[0].stat.ptsQtrs.length === 1) ||
-                    (i * this.dt > 24 && this.team[0].stat.ptsQtrs.length === 2) ||
-                    (i * this.dt > 36 && this.team[0].stat.ptsQtrs.length === 3)) {
-                this.team[0].stat.ptsQtrs.push(0);
-                this.team[1].stat.ptsQtrs.push(0);
-                this.t = 12;
-                this.recordPlay("quarter");
+        while (true) {
+            while (this.t > 0) {
+                this.simPossessions();
             }
+            quarter += 1;
 
-            // Clock
-            this.t -= this.dt;
-            if (this.t < 0) {
-                this.t = 0;
+            if (quarter === 5) {
+                break;
             }
+            this.team[0].stat.ptsQtrs.push(0);
+            this.team[1].stat.ptsQtrs.push(0);
+            this.t = 12;
+            this.recordPlay("quarter");
+        }
+    };
 
-            // Possession change
-            this.o = (this.o === 1) ? 0 : 1;
-            this.d = (this.o === 1) ? 0 : 1;
+    GameSim.prototype.simOvertime = function() {
+        this.o = 0;
```

Comment:
Should be a jump ball. I guess it should be in regulation too, but that doesn't really matter. It does matter in overtime though. Quick hack is to just randomize initial possession, you don't need to log anything.